M2/cmake/build-libraries.cmake

```diff
@@ -340,28 +340,34 @@ _ADD_COMPONENT_DEPENDENCY(libraries ntl gmp NTL_FOUND)
 
 
 # https://github.com/Macaulay2/flint2
+string(REGEX REPLACE
+  "./configure$" "${CMAKE_SOURCE_DIR}/submodules/flint/configure"
+  flint_CONFIGURE "${CONFIGURE}")
 ExternalProject_Add(build-flint
   PREFIX            libraries/flint
   SOURCE_DIR        ${CMAKE_SOURCE_DIR}/submodules/flint
   BINARY_DIR        libraries/flint/build
-  CMAKE_ARGS        -DCMAKE_INSTALL_PREFIX=${M2_HOST_PREFIX}
-                    -DCMAKE_SYSTEM_PREFIX_PATH=${M2_HOST_PREFIX}
-                    -DCMAKE_MODULE_PATH=${CMAKE_SOURCE_DIR}/cmake
-                    -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-                    -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
-                    -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
-                    -DCMAKE_C_FLAGS=${CFLAGS}
-                    -DCMAKE_CXX_FLAGS=${CXXFLAGS}
-                    -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
-                    -DIPO_SUPPORTED=OFF # TODO: because of clang; see https://github.com/wbhart/flint2/issues/644
-                    -DWITH_NTL=ON
-  INSTALL_COMMAND   ${CMAKE_COMMAND} --install . ${strip_setting}
+  CONFIGURE_COMMAND cd ${CMAKE_SOURCE_DIR}/submodules/flint &&
```

Comment:
I think the right way is to change the `BINARY_DIR` above.

Reply:
I think I'm just going to switch it to an in-tree build.  It turns out that the out-of-tree builds are broken and aren't installing all the header files, which is why the cmake-ubuntu build was failing (factory couldn't find flint.h).  I reported it upstream, so it should be fixed next release.  I'll mention something about bootstrap.sh, too.